### PR TITLE
Usminaff to Development merge

### PR DESCRIPTION
**Pull request by eristow**

### CODE REVIEW

File: modules/users/client/config/users-admin.client.routes.js [hunk 1]
```diff
@@ -2,36 +2,40 @@
...
-        resolve: {
-          userResolve: ['$stateParams', 'Admin', function ($stateParams, Admin) {
-            return Admin.get({
-              userId: $stateParams.userId
-            });
-          }]
-        }
-      });
-  }
+function ($stateProvider) {
```
Comment (angryoctopode) [inline modules/users/client/config/users-admin.client.routes.js]: Indentation here is atrocious.

**Review by angryoctopode — APPROVED** [review]
11/10, would code again.

File: .gitignore [hunk 2]
```diff
@@ -72,3 +72,4 @@ mongod
 *.ntvs*
 *.njsproj
 *.sln
+.vscode/
```
Comment (dkwhite200) [inline .gitignore:75]: Do you even have vscode?

**Review by dkwhite200 — APPROVED** [review]
Coolio

**Review by AriTheGuitarMan — APPROVED** [review]
Seems like a solid work of art.

**Review by wcampbell09 — APPROVED** [review]
approved looks good

**Pull request merged**